plugins/utilities.js

```diff
@@ -0,0 +1,55 @@
+const stylelint = require('stylelint')
+const utilities = require('./lib/primer-utilities')
```

Comment:
For now this is a lib file, but I envision pulling this in from primer/css or primer/primitives